initial-configuration/install-dependencies-docker.sh

```diff
@@ -1,7 +1,18 @@
 
 #!/usr/bin/env bash
 
-CWD=`pwd`
+sed_inplace() {
+  if [ "$(uname)" = "Darwin" ]; then
+    sed -i '' "$@"
+  else
+    sed -i "$@"
+  fi
+}
+
+CWD=$(pwd)
+# this makes tr work on OSX
```

Comment:
What does this variable do on other systems? I assume nothing?

Reply:
nothing bad ... :)